mwdb/model/api_key.py

```diff
@@ -19,6 +19,7 @@ class APIKey(db.Model):
     )
     issued_on = db.Column(db.DateTime, default=datetime.datetime.utcnow, nullable=False)
     issued_by = db.Column(db.Integer, db.ForeignKey("user.id"), nullable=False)
+    name = db.Column(db.String(length=100))
```

Comment:
I would make it non-nullable but allow empty string :thinking:

Reply:
:+1: